.github/workflows/ci.yml

```diff
@@ -9,19 +9,19 @@ jobs:
         ocaml-compiler:
           - 4.13.x
           - 4.14.x
+          - 5.0.x
         additional-packages:
-          -
```

Comment:
This line is actually important. It expresses that we should try to compile with **and without** labltk installed...